source/include/FreeRTOS_IP_Utils.h

```diff
@@ -103,6 +103,16 @@ void vPreCheckConfigs( void );
  */
 void prvProcessNetworkDownEvent( struct xNetworkInterface * pxInterface );
 
+#if ( ipconfigIS_ENABLED( ipconfigUSE_IPv6 ) )
+
+/**
+ * @brief Called for IPv6 end-points on network up/down events.
+ * The function manages the MAC address and MLD report associated with the end-point's address.
+ */
```

Comment:
@AniruddhaKanhere I see I forgot the parameters which brings me to a question. Where am I supposed to put the full function description? With the declaration, with the definition, or in both places?
I didn't want to just copy/paste a  large description so I mostly wrote the decription in the .c file.
Thoughts?

Reply:
Oh an I just realized something else. Since `vManageSolicitedNodeAddress()` only deals with IPv6 things, maybe I should have placed it in FreeRTOS_IPv6_Utils.c
Do you agree?

Reply:
I think the description should go in the .c file based on the brief review of +TCP code that I did.

Yes, I agree that we should move the `vManageSolicitedNodeAddress()` to the IPv6_Utils file. What do you think @htibosch?